homework-2/scss/style.scss

```diff
@@ -0,0 +1,1385 @@
+    padding: 8px 0;      
+  }
+  &-list{
+    margin-top: 30px;
+  }
+}
+.slider-logo-single{
+  .slick-slide .single-img{
+    max-width: 235px;
+    width: 100%;
+    margin: 0 auto;
+  }
+  .slick-disabled{
+    opacity: 0.5;
+  }
+}
+.slider-logo-list{
+  margin-top: 40px;
+  .img-logo{
+    max-width: 51px;
+    width: 100%;
+  }
+}
+.slick-dots{
+  display: flex;
+  width: 100%;
+  li{    
+    flex: 1;
+    background: #E5E5E5;
+    height: 5px;
+
+    &.slick-active{
+      background: #22bf95;
+    }
+    button{
+      visibility: hidden;
+    }
+  }
+}
+.slider-logo{
+  padding: 100px 0; 
+}
+.description{
+  max-width: 315px;
+  width: 100%;
+  &-text{
+    text-align: center;
+  }
+  &-title{
+    margin-top: 26px;
+    text-align: center;
+  }
+}
+.services{
+  padding-bottom: 40px;
+  &-item-text{
+    padding: 0 20px;
+  }
+  &-text{
+    margin-top: 20px;
+    font-size: 13px;
+    line-height: 24px;
+  }
+  &-img{
+    margin: 30px auto 0;
+  }
+}
+.get-more{
+  margin-top: -60px;
+  padding: 19px 25px 30px;
+  background-image: $img-bg-card;
+  border-radius: 5px;
+  filter: drop-shadow(0px 12px 24px #e5e5e5);
+  &-title{
+    padding-bottom: 5px;
+    color: $color-get-more-title;
+    font-family: $font-title, sans-serif;
+    font-size: 10px;
+    line-height: 24px;
+    letter-spacing: 1.6px;
+    text-transform: uppercase;
+    border-bottom: 1px solid rgba(255, 255, 255, 0.5);
+  }
+  &-link{
+    margin: 34px 0 0 0px;   
+    .get-link{
+      color: $color-get-more-title;
+      font-size: 16px;
+      line-height: 24px;
+    }
+  }
+  .get-item{
+    margin-top: 40px;
+  }
+}
+.get-text{
+  position: relative;
+  &::after{
+    content: '';
+    position: absolute;
+    display: inline-block;
+    width: 35px;
+    height: 35px;
+    margin: -7px 0 0 89px;
+    background-color: transparent;
+    border: 1px solid #fff;
+    border-radius: 50%;
+  }
+}
+.card-item-wrapper{
+  position: relative;
+  display: flex;
+  align-items: center;
+  padding: 9px 0 16px;
+  &::before{
+    content: "";
+    position: absolute;
+    top: 0;
+    width: 218px;
+    height: 2px;
+    background: #f2f2f2;
+  }
+}
+.card-item{
+  position: relative;
+  margin-left: 20px;
+  color: $color-description-text;
+  font-size: 13px;
+  line-height: 24px;
+  &-minus{
+    color: #e5e5e5;
+  }
+}
+.check-icon{
+  position: relative;
+  display: inline-block;
+  width: 35px;
+  height: 35px;
+  background: transparent;
+  border: 2px solid #22bf95;
+  border-radius: 50%;
+  &::before{
+    content: "";
+    width: 2px;
+    background: #22bf95;
+    height: 9px;
+    position: absolute;
+    top: 14px;
+    transform: rotate(138deg);
+    left: 9px;
+  }
+  &::after{
+    content: "";
+    width: 2px;
+    background: #22bf95;
+    height: 18px;
+    position: absolute;
+    top: 7px;
+    transform: rotate(-138deg);
+    left: 18px;
+  }
+}
+.minus-icon{
+  position: relative;
+  display: inline-block;
+  width: 35px;
```

Comment:
this will not work correctly
![image](https://user-images.githubusercontent.com/17281920/121732524-09814180-cafb-11eb-8fe3-60f3598521aa.png)

Reply:
Actually - you do not need flex on parent element. You can add padding to parent and position icon on absolute

Reply:
I will fix bug in this slider.

Reply:
done